### ISSUE TITLE
Annual stats report

### ISSUE THREAD

**Issue by bobintetley** [bobintetley/asm3#865]
While the annual figures report is good for breaking down animals in various ways, it would be nice to have a stats dashboard type view that covered multiple areas and used subreports so we could add sections in future. It should also be able to be run for multiple years.

This has come out of a user request where they sent a spreadsheet.

[CARE Stats 2007-2014.xlsx](https://github.com/bobintetley/asm3/files/4785431/CARE.Stats.2007-2014.xlsx)

The proposed sections are:

Intake by year (list of years the period covers with average at bottom)
Intake by species (list of species in period)
Intakes by jurisdiction
Intakes by city
Dogs and cats by sex (counts of male/female for speciesid=1/2)
Adoptions by jurisidiction
Adoptions by city
Total outcomes

All of them will use PCT to show a percentage column so they're outputting in the group footer.

The jurisdiction reports are blocked by #650
